mailer/engine.py

```diff
@@ -103,7 +102,7 @@ def release_lock(lock):
     logging.debug("released.")
 
 
-def send_all():
+def send_all():  # noqa: C901  # TODO: refactor to reduce complexity
```

Comment:
I'm personally not that worried about complexity here - I don't think clarity would be increased that much by refactoring it.